website/components/TypeDefinition.tsx

```diff
@@ -14,7 +15,11 @@ function TypeDefinition({ markdownAst, readme }) {
     .map(treeItem => treeItem.children?.[0].value);
 
   // Gets types defined in readmes to expand upon below the prop table
-  const interfaceDefinitions = readme?.match(/(?<=typescript).*?(?=```)/gs);
+  const getTypes: Array<string> = readme?.split('typescript');
```

Comment:
```suggestion
  const getTypes: Array<string> = readme.split('typescript');
```

Also shouldn't need the explicit type if you add a type for the props

Reply:
Yep got it